contracts/src/finance/vesting_wallet.rs

```diff
@@ -89,15 +91,11 @@ pub enum Error {
     InvalidToken(InvalidToken),
 }
 
-pub use token::IErc20;
-mod token {
-    #![allow(missing_docs)]
-    #![cfg_attr(coverage_nightly, coverage(off))]
-    stylus_sdk::stylus_proc::sol_interface! {
-        /// Interface of the ERC-20 token.
-        interface IErc20 {
-            function balanceOf(address account) external view returns (uint256);
-        }
+sol_interface! {
+    /// Interface of the ERC-20 token.
+    #[allow(missing_docs)]
+    interface IErc20 {
+        function balanceOf(address account) external view returns (uint256);
```

Comment:
Won't coverage be reduced by including this?

Reply:
Yeap. It will be reduced, currently the simplest way to compile it.

Reply:
It doesn't compile if you do:
```rust
pub use token::IErc20;
mod token {
    #![allow(missing_docs)]
    #![cfg_attr(coverage_nightly, coverage(off))]
    stylus_sdk::prelude::sol_interface! {
        /// Interface of the ERC-20 token.
        interface IErc20 {
            function balanceOf(address account) external view returns (uint256);
        }
    }
}
```
? :open_mouth:

Reply:
`IErc20` now made private. So it will de-sugar to `struct IErc20`, instead of `pub struct IErc20`. And visibility level will just mod wise

Reply:
May be try to change it again in stylus sdk..